src/inmanta/server/services/resourceservice.py

```diff
@@ -412,6 +413,11 @@ async def mark_deployed(
         :param version: Version of the resources to consider.
         :param filter: Filter function that takes a resource id as an argument and returns True if it should be kept.
         """
+        # TODO: can we get rid of mark deployed now????
+        # Looks feasible on the path for local dependencies, with start deploy
+        # what about CAD's?
+        # what about the dashboard?
```

Comment:
If you mean the web-console with "dashboard", I *think* it doesn't rely on `mark_deployed` at all.